src/metax_api/models/catalog_record.py

```diff
@@ -438,8 +438,14 @@ def user_is_privileged(self, request):
         :return:
         """
         if request.user.is_service:
-            # knows what they are doing
-            return True
+            if request.method == 'GET':
+                if not self._check_catalog_permissions(self.data_catalog.catalog_record_group_read,
+                self.data_catalog.catalog_record_services_read, request):
```

Comment:
Here should be indentation (two tabs)

Reply:
Ok fixed

Reply:
....Flake8 complains now that these lines are over-intended